Notebooks/OOPS_TMT_Data_Processing_And_QC.Rmd

```diff
@@ -37,11 +35,31 @@ crap_accessions <- crap_fasta %>%
   unlist()
 ```
 
+No cRAP proteins are removed! Have they already been removed before the file was 
```

Comment:
Appears that there are no contaminant proteins in the output from PD. This is unusual, but perhaps Bini has switched on an option to ensure this? We should check with Bini.